File: coding-challenge-1/src/models/member.ts
/**
 * Represents a team member in the rotation system
 */
export interface Member {
  id: number;
  name: string;
  isActive: boolean;
}

/**
 * Factory function to create a Member
 */
export function createMember(id: number, name: string, isActive: boolean = true): Member {
  return { id, name, isActive };
}

File: coding-challenge-1/TeamRotator/Member.cs
namespace TeamRotator;

public class Member
{
    public Member(int id, string name, bool isActive = true)
    {
        Id = id;
        Name = name;
        IsActive = isActive;
    }

    public int Id { get; }
    public string Name { get; }
    public bool IsActive { get; }
}